tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -2295,194 +2360,231 @@ <h2>Create a distance feature</h2>
         if (!this.inferenceStats_ || !this.visdata ||
             this.selectedLabelFeature == '' ||
             this.examplesAndInferences.length == 0 ||
-            this.examplesAndInferences[0].inferences == null) {
+            this.examplesAndInferences[0].inferences == null ||
+            this.examplesAndInferences[0].inferences[0].length
+              != this.numModels ||
+            (this.inferenceStats_.length != this.numModels &&
```

Comment:
maybe add comments about this check, as its gotten complex with these new cases

Reply:
added description.